bagelbids/cli.py

```diff
@@ -155,9 +172,58 @@ def bids(
 
     # Strip and store context to be added back later, since it's not part of
     # (and can't be easily added) to the existing data model
-    context = jsonld.pop("@context")
+    context = {"@context": jsonld.pop("@context")}
 
     try:
         pheno_dataset = models.Dataset.parse_obj(jsonld)
     except ValidationError as err:
         print(err)
+
+    pheno_subject_list = []
+    for pheno_subject in getattr(pheno_dataset, "hasSamples"):
+        pheno_subject_list.append(pheno_subject.label)
+    bids_subject_list = ["sub-" + sub_id for sub_id in layout.get_subjects()]
+
+    check_unique_bids_subjects(
+        pheno_subjects=pheno_subject_list, bids_subjects=bids_subject_list
+    )
+
+    for bids_sub_id in layout.get_subjects():
+        for pheno_subject in pheno_dataset.hasSamples:
+            if f"sub-{bids_sub_id}" == pheno_subject.label:
+                session_list = []
+                # For some reason .get_sessions() doesn't always follow alphanumeric order
+                # By default (without sorting) the session lists look like ["02", "01"] per subject
```

Comment:
I don't think it makes a functional difference. Without this sorting though, for a subject in the resulting pheno_bids.jsonld the first entry in `hasSession` may have a session label of "02" instead of "01", which is a little unusual - at first glance I would think that the first session is missing (except it just comes after "02" in the list).